plugins/node/instrumentation-dataloader/src/instrumentation.ts

```diff
@@ -73,8 +73,7 @@ export class DataloaderInstrumentation extends InstrumentationBase {
             this._unwrap(dataloader.prototype, 'loadMany');
           }
         }
-        // cast it to module definition of unknown type to avoid exposing Dataloader types on public APIs
-      ) as InstrumentationNodeModuleDefinition<unknown>,
```

Comment:
Since this comment is here, I think it's not safe to type the `moduleExports` with the now-removed generic type, which does not give a lot of value IMO.